google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudApiManagementConfirmationDialog.java

```diff
@@ -153,6 +195,11 @@ private void createUIComponents() {
     roleTable = new ServiceAccountRolesTable(roles);
   }
 
+  private static boolean isValidDirectory(String path) {
+    File file = new File(path);
+    return file.exists() && file.isDirectory() && file.canWrite();
```

Comment:
We could consider slightly shorter option with `Files.isWritable(path)` that combines all calls together.

Reply:
I thought of that, but that takes in a `Path` which would we need to create via `Paths#get` which then leads to catching exceptions. Do you still think its better? or is there a shorter way you were thinking?

Reply:
Oh you mean a user could enter a total nonsense and `Paths.get` would throw an exception? Yeah, then probably it's easier to read without exceptions.

Reply:
yea exactly. ok I'll leave it then